modules/mobile/spec/request/appointment_request_messages_spec.rb

```diff
@@ -0,0 +1,59 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+require_relative '../support/iam_session_helper'
+
+RSpec.describe 'appointment_request_messages', type: :request do
+  before do
+    allow_any_instance_of(IAMUser).to receive(:icn).and_return('1012845331V153043')
+    iam_sign_in(build(:iam_user))
+    allow_any_instance_of(VAOS::UserService).to receive(:session).and_return('stubbed_token')
+  end
+
+  describe '#index' do
+    let(:request_id) { '8a4886886e4c8e22016e5bee49c30007' }
+
+    it 'returns 200 with message details upon success' do
+      VCR.use_cassette('vaos/messages/get_messages', match_requests_on: %i[method uri]) do
```

Comment:
AFAICT, we always make copies of vcr cassettes from other modules. I haven't done that here because I wouldn't have changed anything about the cassettes. In other cases, I've made copies because I needed them to have different params or return a more limited data set. Given that the cassettes are directly tied to the vaos service we use for this endpoint, it makes sense to continue using their cassettes. I can change this if we would rather make our own copies as a policy.

Reply:
I would make a copy as well in this case just so we're in control of the resources for our specs.

Reply:
I wasn't sure if we do this by policy or not. Sounds like it's policy. Thanks for clarifying.